src/main/resources/assets/immersiveengineering/lang/de_DE.lang

```diff
@@ -0,0 +1,312 @@
+item.ImmersiveEngineering.material.coalCoke.name=Koks
+item.ImmersiveEngineering.material.gunpartBarrel.name=Revolver Lauf
+item.ImmersiveEngineering.material.gunpartDrum.name=Revolver Trommel
+item.ImmersiveEngineering.material.gunpartGrip.name=Revolver Griff
+item.ImmersiveEngineering.material.gunpartHammer.name=Revolver Abzug
+item.ImmersiveEngineering.material.componentIron.name=Eisen Mechanischer Bestandteil
+item.ImmersiveEngineering.material.componentSteel.name=Stahl Mechanischer Bestandteil
+item.ImmersiveEngineering.seed.hemp.name=Industrieller Hanf Samen
+
+item.ImmersiveEngineering.fluidContainers.bottleCreosote.name=Teeröl Flasche
+item.ImmersiveEngineering.fluidContainers.bucketCreosote.name=Teeröl Eimer
+item.ImmersiveEngineering.fluidContainers.bottlePlantoil.name=Pflanzen Öl Flasche
+item.ImmersiveEngineering.fluidContainers.bucketPlantoil.name=Pflanzen Öl Eimer
+item.ImmersiveEngineering.fluidContainers.bottleEthanol.name=Ethanol Flasche
+item.ImmersiveEngineering.fluidContainers.bucketEthanol.name=Ethanol Eimer
+item.ImmersiveEngineering.fluidContainers.bottleBiodiesel.name=Bio Diesel Flasche
+item.ImmersiveEngineering.fluidContainers.bucketBiodiesel.name=Bio Diesel Eimer
+
+item.ImmersiveEngineering.coil.copper.name=Kupfer Kabel Spule
+item.ImmersiveEngineering.coil.electrum.name=Elektrum Kabel Spule
+item.ImmersiveEngineering.coil.HV.name=Hoch-Spannungs Kabel Spule
+item.ImmersiveEngineering.coil.rope.name=Hanf Leinen Rolle
+item.ImmersiveEngineering.coil.structural.name=Sahl Kabel Spule
+item.ImmersiveEngineering.tool.hammer.name=Ingenieurs Hammer
+item.ImmersiveEngineering.tool.wirecutter.name=Ingenieurs Kabelschneider
+item.ImmersiveEngineering.tool.voltmeter.name=Ingenieurs Messgerät
+item.ImmersiveEngineering.tool.manual.name=Ingenieurs Handbuch
+
+item.ImmersiveEngineering.revolver.normal.name=Revolver
+item.ImmersiveEngineering.revolver.elite.name=Elite Revolver
+item.ImmersiveEngineering.revolver.speedloader.name=Revolver Schnelllader
+item.ImmersiveEngineering.revolver.sns.name=Elite Revolver
+item.ImmersiveEngineering.revolver.nerf.name=Nerf'd Revolver
+item.ImmersiveEngineering.revolver.oblivion.name=Oblivion
+item.ImmersiveEngineering.revolver.oathkeeper.name=Oathkeeper
+item.ImmersiveEngineering.revolver.bee.name=Beevolver
+item.ImmersiveEngineering.revolver.fenrir.name=Fenrir
+
+item.ImmersiveEngineering.bullet.emptyCasing.name=Leeres Gehäuse
+item.ImmersiveEngineering.bullet.emptyShell.name=Leere Hülse
+item.ImmersiveEngineering.bullet.casull.name=Casull Ladung
+item.ImmersiveEngineering.bullet.armorPiercing.name=Rüstungsdurchbrechende Ladung
+item.ImmersiveEngineering.bullet.buckshot.name=Schrot Ladung
+item.ImmersiveEngineering.bullet.HE.name=High-Explosive Ladung
+item.ImmersiveEngineering.bullet.dragonsbreath.name=Drachenatem Ladung
+
+#CREATIVE
+itemGroup.ImmersiveEngineering=Immersive Engineering
+
+#DESCRIPTIONS
+desc.ImmersiveEngineering.info.attachedTo=Verbindet von %1$s, %2$s, %3$s
```

Comment:
Are you sure you translated this yourself? This sounds like word-by-word google translate...
Should have been `Verbunden mit`